scripts/test-triton-models.py

```diff
@@ -29,4 +45,37 @@ async def main():
         except:
             print("Model test failed: ", ret["result"]["rdf"]["name"], ret["result"]["rdf"].get("id"))
 
+    # One-off test of the cellpose model
+    image = imageio.v3.imread('https://static.imjoy.io/img/img02.png')
+    try:
+        ret = await triton.execute(
+            inputs=[image.astype('float32'), {'diameter': 30}],
+            model_name="cellpose-python",
+            decode_json=True
+        )
+        mask = ret['mask'][0]
+        print("Model test passed: ", "Cellpose using cellpose-python")
```

Comment:
Can we add some assert statements here to check the mask shape etc.? There should be a `success` key, and if it's false, there should be an `error` key to get the stack trace.

Reply:
Shouldn't the mask access fail if the process failed?

Reply:
But yeah, can check for a success key and get it's value too

Reply:
There is no success field... the return value is: 
```python
{
'mask': array([[0, 0, 0, ..., 0, 0, 0],
       [0, 0, 0, ..., 0, 0, 0],
       [0, 0, 0, ..., 0, 0, 0],
       ...,
       [0, 0, 0, ..., 0, 0, 0],
       [0, 0, 0, ..., 0, 0, 0],
       [0, 0, 0, ..., 0, 0, 0]], dtype=int32), 
 '__info__': {
     'model_name': 'stardist', 
     'model_version': '1', 
     'outputs': [
         {'name': 'mask', 
           'datatype': 'INT32', 
           'shape': [349, 467], 
           'parameters': {'binary_data_size': 651932}
          }
       ]
    }
}
```

Reply:
So checking that the mask exists and contains a 2d array is probably about as good a check as we can do. Presumably it should have the same shape as the input?

NB: this is stardist not the bioengine-model-runner

Reply:
Just added shape tests... `stardist` was fine, but `cellpose-python` returns a stupid transposed array :roll_eyes: :+1: 
```
 (1, 349, 467) != (349, 467, 3)
 ```

Reply:
Actually, ignore that. Upon closer paying attention, I get it :rofl: . The input is RGB, output is binary mask (and for some reason has to have the singleton dimension added).